README.md

```diff
@@ -1,2 +1,351 @@
-# year-cohort-projectName (eg: 22-jul-projectName)
-project template repo
+# Recipes API
+
+## API documentation
+
+### Return/Create user userId - story-2
+
+- **URL**
+
+  /users
+
+- **Method:**
+
+  `POST`
+
+- **URL Params**
+
+  none
+
+  **Example:**
+
+  `/users`
+
+## get user
```

Comment:
example of what the frontend is going to send you?

Reply:
done!